src/utils/utils_log.c

```diff
@@ -32,6 +32,29 @@
 #include "utils_common.h"
 #include "utils_log.h"
 
+#define UMF_VERSION_MAGIC_STR "\x00@(#) "
+#define UMF_VERSION_PREF_STR "Intel(R) "
+#define UMF_VERSION_PREFIX UMF_VERSION_MAGIC_STR UMF_VERSION_PREF_STR
```

Comment:
all these 3 defines have "VERSION" in it, but they are not only used with version string

Reply:
Removed